### PR TITLE
Docs - Improve SEO

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Added keywords, image and description meta tags to all docs
* Renamed and moved docs files for browser friendly descriptive urls
* Fixed typos
* Fixed title capitalization
* Hosting images locally
* Updated links to wiki

## Links to any relevant issues

* Resolves  iotaledger/docs.iota.org#38

## Type of change

- [x] Enhancement (a non-breaking change which adds functionality)
- [x] Documentation Fix

## How the change has been tested

Docs site was built and served locally using node 12.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@iotaledger/tech-writers  could someone review this ?

**Review by huhn511 — APPROVED** [review]
LGTM! 👍

**Review by apenzk — APPROVED** [review]

File: docs/1.1Introduction.md [hunk 1]
```diff
@@ -1,56 +1,55 @@
...
+- access mana
+- layers
+- communication
+- networks
+---
+
+# 1.1 Introduction
+
+:::info
+These specifications are for a research prototype and are a work in progress, thus they may change.  Please refer to the preface for a description of their exact status.
```
Comment (aleksei-korolev) [inline docs/1.1Introduction.md:19]:
```suggestion
These specifications are in progress and describe a research prototype. They will change — refer to the preface for updates.
```

File: docs/2.3StandardPayloadsLayout.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
+---
+description: Payloads may recursively contain other payloads, that enables the creation of higher-level protocols based on the same concepts of layers, as in traditional software and network architecture.
```
Comment (aleksei-korolev) [inline docs/2.3StandardPayloadsLayout.md:2]:
```suggestion
description: This page describes the Standard Payloads Layout and different types of payloads that it includes.
```

Btw, I suggest my changes assuming that this is a description of what you find inside the page, not description of the page's subject.

File: docs/3.3PeerDiscovery.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
+---
+description: This section defines the Peer Discovery protocol used to expose an interface providing a list of all the verified peers, its logic and the different requests and responses exchanged.
```
Comment (aleksei-korolev) [inline docs/3.3PeerDiscovery.md:2]:
```suggestion
description: This section defines the Peer Discovery protocol that you use to receive a list of verified peers. Here we explain how it works and what kind of requests and responses you could use.
```

File: docs/3.4NeighborSelection.md [hunk 4]
```diff
@@ -1,26 +1,40 @@
+---
+description: In order for the network to work efficiently and for the nodes to be kept up-to-date about the ledger state, nodes exchange information with each other. Each node establishes a communication channel with a small subset of nodes (i.e., neighbors) via a process called `peering`.
```
Comment (aleksei-korolev) [inline docs/3.4NeighborSelection.md:2]:
```suggestion
description: This page explains how nodes find their neighbors and connect to them, a process that we call "peering". It allows nodes work efficiently without disrupting the actual ledger state.
```

File: docs/4.2Timestamps.md [hunk 5]
```diff
@@ -1,17 +1,32 @@
+---
+description: While online nodes may leverage FPC to vote on timestamps, nodes that join the network at a later time use an approach based on the approval weight to determine the validity of timestamps.
```
Comment (aleksei-korolev) [inline docs/4.2Timestamps.md:2]:
```suggestion
description: Timestamps is a tool that allows nodes to keep track of time and agree on the age of the messages. We combine synchronious and asynchronious approaches. While online nodes vote on timestamps to achieve consensus, nodes that join the network later use approval weights.
```

File: docs/4.3TipSelectionAlgorithm.md [hunk 6]
```diff
@@ -1,3 +1,14 @@
+---
+description: The tip selection algorithm is the method by which messages are selected for approval by other issued messages joining the network. It allows the Tangle to grow in a stable and secure way, with quick approval and finality times
```
Comment (aleksei-korolev) [inline docs/4.3TipSelectionAlgorithm.md:2]:
```suggestion
description: The tip selection algorithm picks messages for approval by messages that join the network later. It allows the Tangle to grow in a stable and secure way, with quick approval and finality times.
```

File: docs/4.5RateControl.md [hunk 7]
```diff
@@ -1,19 +1,31 @@
+---
+description: In order to enable the machine-to-machine economy, IOTA does not allow either mining race nor fees, which makes an explicit rate control mechanism necessary to act as an emergency br during spam attacks, by slowing down the rate of messages a node can issue.
```
Comment (aleksei-korolev) [inline docs/4.5RateControl.md:2]:
```suggestion
description: IOTA has no mining races or fees. To protect itself against spam attacks, Tangle has a rate control mechanism that could slow down the rate of messages from malicious nodes.

File: docs/4.6CongestionControl.md [hunk 8]
```diff
@@ -1,3 +1,14 @@
+---
+description: test
```
Comment (aleksei-korolev) [inline docs/4.6CongestionControl.md:2]: Bruh.

File: docs/5.2LedgerState.md [hunk 9]
```diff
@@ -1,9 +1,21 @@
+---
+description: The Realities Ledger State deals with double spends and leverages on certain properties of UTXO.
```
Comment (aleksei-korolev) [inline docs/5.2LedgerState.md:2]:
```suggestion
description: The Realities Ledger State prevents double spending by leveraging certain properties of UTXO.  
```

File: docs/5.2LedgerState.md [hunk 10]
```diff
@@ -1,9 +1,21 @@
+---
+description: The Realities Ledger State deals with double spends and leverages on certain properties of UTXO.  
+image: /img/logo/Coordicide_Logo_Black.png
+keywords:
+- transactions
```
Comment (aleksei-korolev) [inline docs/5.2LedgerState.md:5]:
```suggestion
- transactions
- double spending
```

File: docs/5.3Mana.md [hunk 11]
```diff
@@ -1,23 +1,35 @@
-# 5.3 Mana
+---
+description: This section introduces  *access Mana* and *consensus Mana* and determines which modules use each of these two types of Mana.
```
Comment (aleksei-korolev) [inline docs/5.3Mana.md:2]:
```suggestion
description: This section introduces  *access Mana* and *consensus Mana* and explains which modules use which Mana. 
```

File: docs/6.1ObjectsOfConsensus.md [hunk 12]
```diff
@@ -1,10 +1,19 @@
+---
+description: asd 
+image: FPC is a binary voting protocol where each node starts with an initial opinion (a nulled boolean) on an objec. This specification describes how this binary voting protocol interacts with the rest of the IOTA protocol.
```
Comment (aleksei-korolev) [inline docs/6.1ObjectsOfConsensus.md:3]: .

File: docs/6.3FastProbabilisticConsensus.md [hunk 13]
```diff
@@ -1,8 +1,21 @@
+---
+description: Probabilistic Consensus (FPC), is triggered if the eligibility of a  message or transaction object is uncertain.
```
Comment (aleksei-korolev) [inline docs/6.3FastProbabilisticConsensus.md:2]:
```suggestion
description: Fast Probabilistic Consensus (FPC) checks if a message or a transaction is eligible when it is not yet certain.
```

File: docs/6.4Finalization.md [hunk 14]
```diff
@@ -1,36 +1,49 @@
...
+image: /img/research-specifications/approval-weight.png
+keywords:
+- approval weight
+- finality
+- node
+- message
+- branch
+- ledger state
+- active consensus mana
+-
```
Comment (aleksei-korolev) [inline docs/6.4Finalization.md:12]:
```suggestion
```

File: docs/6.5dRNG.md [hunk 15]
```diff
@@ -1,3 +1,16 @@
+---
+description: asd
```
Comment (aleksei-korolev) [inline docs/6.5dRNG.md:2]: .

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: docs/6.5dRNG.md [hunk 16]
```diff
@@ -1,3 +1,16 @@
+---
+description: The Distributed Random Number Generator protocol is divided into three phases -  Committee selection, DKG and publication.
```
Comment (aleksei-korolev) [inline docs/6.5dRNG.md:2]:
```suggestion
description: The Distributed Random Number Generator protocol is divided into three phases — committee selection, DKG and publication. 
```

**Review by aleksei-korolev — APPROVED** [review]

**Pull request merged**